code/game/turfs/open.dm

```diff
@@ -145,7 +145,7 @@
 			C.slipping = TRUE
```

Comment:
Setting slipping to false afterwards is the same as just changing this line to C.slipping = FALSE and removing the line below outright